extensions/powertoys-tool-runner/src/utils/triggerEvent.ts

```diff
@@ -0,0 +1,23 @@
+import { runPowerShellScript, showFailureToast } from "@raycast/utils";
+
+/**
+ * Triggers a PowerToys Windows event using Raycast's PowerShell utility
+ */
+export async function triggerPowerToysEvent(eventName: string, toolName: string): Promise<void> {
+  try {
+    const psCommand = `[System.Threading.EventWaitHandle]::OpenExisting('${eventName}').Set()`;
+
+    console.log(`Triggering ${toolName}...`);
+
+    await runPowerShellScript(psCommand, {
+      timeout: 5000,
+    });
+
+    console.log(`${toolName} triggered successfully`);
+  } catch (error) {
+    console.error(`${toolName} launch failed:`, error);
+    await showFailureToast(error, {
+      title: `Failed to launch ${toolName}`,
```

Comment:
Thanks for the PR! do you think we can show a nicer custom message when it errors because the user doesn't have PowerToys installed?